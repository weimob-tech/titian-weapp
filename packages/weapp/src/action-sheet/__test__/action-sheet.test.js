const { getComponent } = require('../../common/test/getComponent');

describe('test component', () => {
  it('should render success', () => {
    const comp = getComponent('action-sheet');
    expect(comp).toMatchSnapshot();
  });
});
