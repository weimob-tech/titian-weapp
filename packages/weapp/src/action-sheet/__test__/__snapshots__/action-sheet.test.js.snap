// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test component should render success 1`] = `
<main>
  <ti-popup
    extClass="titian-action-sheet"
    bind:close="onClose"
  >
    <ti-transition
      enterDoneClass="titian-popup-enter-done"
      exitClass="titian-popup-enter-done"
      extClass="titian-popup-mask ext-mask-class"
      preventScroll="{{true}}"
      bind:tap="onTapMask"
      catch:touchmove="noop"
    />
    <ti-transition
      enterActiveClass="titian-popup-slide-up-start-active titian-popup"
      enterClass="titian-popup-slide-up-start titian-popup"
      enterDoneClass="titian-popup-slide-up-middle titian-popup"
      exitActiveClass="titian-popup-slide-up-end-active titian-popup"
      exitClass="titian-popup-slide-up-middle titian-popup"
      exitDoneClass="titian-popup-slide-up-end titian-popup"
      extClass="ext-content-class titian-popup-bottom titian-popup"
      bind:enter="onEnter"
      bind:entered="onEntered"
      bind:exit="onExit"
      bind:exited="onExited"
      catch:touchmove="noop"
    />
  </ti-popup>
</main>
`;
